example-proxy.ini

```diff
@@ -1,8 +1,6 @@
 [privacyidea]
 # URL of your privacyIDEA installation
 instance = http://10.0.0.1
-# Realm to use for authentication (may also be left blank)
-realm =
```

Comment:
Yep! I thought it would be nice to have a "realm mapping" step analogously to the "user mapping" step.
Having one default realm could be achieved by the following config then:
```
[realm-mapping]
strategy = static
realm =
```

... so this is really just a modification of the config file layout.